active/dceu.md

```diff
@@ -20,20 +27,22 @@ Delegated responsibilities TBD.
                          
 ## Future membership
 
-Membership is open to former organizers of DjangoCon EUs. Questions about membership can be directed to european-organizers-support@djangoproject.com. [Contact the board](https://www.djangoproject.com/contact/foundation/) to express interest.
+Membership is open to former organizers of DjangoCon EUs. Questions about membership can be directed to european-organizers-support@djangoproject.com.
 
 Membership is self-managed: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
 
 ## Budget
 
-TBD
+No budget needed at this time.
 
 ## Comms
 
-The team will operate via a Google Group, reachable via european-organizers-support@djangoproject.com. There might be additional more synchronous communication (e.g. Slack or Discord) if needed.
+The team internally operates via a Google Group. This is also the primary external point of contact to reach the working group: european-organizers-support@djangoproject.com
+
+The working group is also present in the #europe channel on DSF slack as a public place of discussion.
```

Comment:
This seems ok to me but do keep in mind the DSF Slack is only for the inner circle of people who are currently active with DSF working groups / Django teams / DSF-sponsored initiatives.

Reply:
Ah, wasn't sure about that. Should we rather remove this line then?

Reply:
I’m happy either way